snp_motion_planning/src/planner_profiles.hpp

```diff
@@ -8,6 +8,10 @@
 #include <tesseract_motion_planners/trajopt/profile/trajopt_default_composite_profile.h>
 #include <tesseract_motion_planners/simple/profile/simple_planner_lvs_plan_profile.h>
 
+#include <tesseract_task_composer/profiles/contact_check_profile.h>
+#include <tesseract_task_composer/profiles/min_length_profile.h>
+#include <tesseract_task_composer/profiles/interative_spline_parameterization_profile.h>
```

Comment:
I don't think this file needs any task composer includes because it only deals with motion planner profiles

Reply:
These (except for the contact check one) are technically used in the planning_server.cpp file but I put the includes here because they're profiles. It's a little confusing though because all the other profiles for motion planning are in the tesseract_motion_planners package. We can drop the contact check profile and move the others to the planning_server.cpp file though if you want me to